lib/diagnostic.js

```diff
@@ -47,7 +47,7 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.fire.bind(alderaan);
```

Comment:
Close. `.bind` sets the context of `fire` to `alderaan`, but this doesn't _invoke_ `fire`. You need to do that in order to actually 'fire on alderaan'.

Also, less significantly `fire` returns a value, and this value isn't being returned from `fireOnAlderaan`. Returning a value wasn't specified in the directions, but since we can see `fire` and we know that all it does is return a value, we probably would want to return that value from `fireOnAlderaan` as well - otherwise, it just goes off into space (as it were).